docs/docs/api-reference/starlark-reference/toleration.md

```diff
@@ -8,10 +8,32 @@ can be used with a [ServiceConfig][service-config] object.
 
 ```python
 toleration = Toleration(
+    # key is the taint key that the toleration applies to. Empty means match all taint keys.
+    # If the key is empty, operator must be Exists; this combination means to match all values and all keys.
+    # OPTIONAL
     key = "key",
+
+    # operator represents a key's relationship to the value.
+    # Valid operators are Exists and Equal. Defaults to Equal.
+    # Exists is equivalent to wildcard for value, so that a pod can tolerate all taints of a particular category.
+    # OPTIONAL
     operator = "Equal",
+
+	# value is the taint value the toleration matches to.
```

Comment:
Extra indentation here and in the following fields' comments

Reply:
put tabs now! does it look better?